docs/guides/migrate-to-zeebe-user-tasks.md

```diff
@@ -0,0 +1,376 @@
+
+Use the following decision helper questionaire to figure out what's right for you.
+
+<FormViewer schema={ userTaskMigrationDecisionHelperForm } customStyles={{
+    border: "none",
+    padding: "0px",
+    maxWidth: "unset",
+    background: "transparent",
+    width: "calc(100% + 42px)",
+    marginLeft: "-28px"
+}} />
+
+### Task type differences
+
+Learn the differences between both task types and make an informed decision, and understand the new capabilities of Zeebe user tasks. Refer to this table for important high-level differences of the two task types:
+
+<table>
+    <tr>
+        <th></th>
+        <th>
+            <div>Job-worker based user tasks</div>
+            <TableTextSmall>Existing implementation</TableTextSmall>
+        </th>
+        <th>
+            <div>Zeebe user tasks</div>
+            <TableTextSmall>Recommended for new projects</TableTextSmall>
+        </th>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Implementation location</th>
+        <td>Tasklist</td>
+        <td>
+            <div>Zeebe</div>
+            <TableTextSmall>Does not require Tasklist web app to run</TableTextSmall>
+        </td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Compatible versions</th>
+        <td>8.0 +</td>
+        <td>8.5 +</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports Tasklist UI</th>
+        <td><YesItem /></td>
+        <td><YesItem /></td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "center" }} colspan={3}>API</th>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports Tasklist API</th>
+        <td>
+            <YesItem />
+            <TableTextSmall>Full support</TableTextSmall>
+        </td>
+        <td>
+            <div>Partially</div>
+            <TableTextSmall>Queries, GET Tasks, Forms, Variables</TableTextSmall>
+            <TableTextSmall>ℹ  Note you have to use Zeebe and Tasklist APIs to use Zeebe User Tasks at the moment</TableTextSmall>
+        </td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports Zeebe API</th>
+        <td><NoItem /></td>
+        <td>
+            <YesItem />
+            <TableTextSmall>Task state operations (assign/update/complete)</TableTextSmall>
+        </td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports job workers</th>
+        <td><YesItem /></td>
+        <td><NoItem /></td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports task lifecycle events</th>
+        <td>
+            <NoItem />
+            <TableTextSmall>Basic only: created/completed/canceled</TableTextSmall>
+        </td>
+        <td>
+            <YesItem />
+            <TableTextSmall>Full lifecycle events incl. custom actions</TableTextSmall>
```

Comment:
by incl. and excl. do you mean including and excluding? I would probably just write it out in full.